src/line.rs

```diff
@@ -49,30 +49,51 @@ impl Line {
             warn!("Line must have at least 1 points to render");
             return;
         }
+      
+        let stroke = tiny_skia::Stroke {
+            width: self.width.0,
+            ..Default::default()
+        };
+        let mut builder = tiny_skia::PathBuilder::new();
+        let Some(first_p) = self.points.first() else {
+            return;
+        };
 
-        let width = self.width;
 
-        let (Some(first), Some(last)) = (self.points.first().copied(), self.points.last().copied())
-        else {
+        builder.move_to(first_p.x.0, first_p.y.0);
+        for p in self.points.iter().skip(1) {
+            builder.line_to(p.x.0, p.y.0);
+        }
+        let Some(path) = stroke_path(builder, &stroke, cx) else {
             return;
         };
```

Comment:
Could you feature-gate the code under `tiny-skia` feature? Then include this feature in default. Fallback to the fast approach when the feature is not enabled

Reply:
Later, when the `move_to` method is going to merge, I will update this.